tools/importers/SynchronizeTypographyTokens.spec.ts

```diff
@@ -0,0 +1,115 @@
+import {
+	ImportTypographyVariables,
+	ImportTypographyTokens,
+	requestForBin,
+} from './SynchronizeTypographyTokens';
+const fs = require('fs');
+const tokensFilesConfig = require('./configs/FakeSynchronizeTypographyTokensConfig.json');
+
+describe('Typography Tokens', () => {
+	it('check config & create files', async () => {
+		let requestResponse = await requestForBin(
+			tokensFilesConfig.bin,
+			tokensFilesConfig.jsonBinApiUrl,
+		);
+
+		ImportTypographyVariables(
+			tokensFilesConfig,
+			requestResponse.data.record.values.LMSDesignSystemTypography,
+		);
+
+		ImportTypographyTokens(
+			tokensFilesConfig,
+			requestResponse.data.record.values.LMSDesignSystemTypography,
+		);
+	});
+	it('check variable SCSS file', async () => {
+		await new Promise((r) => setTimeout(r, 500));
+		let variableFileRaw = await fs.promises.readFile(
```

Comment:
does this test depends on previous test? What if we start running tests in parallel?

Reply:
Same answer as here: https://github.com/bethinkpl/design-system/pull/222#discussion_r1023903215

Reply:
This problem still occurs, I mean when I run importer and immediately checking files it always fails. But when I added:
```
await new Promise((r) => setTimeout(r, 1));
```
it's fine. To avoid dependency from tests I put everything in the one test. I couldn't read the file before tests, because only single test can be async.

Reply:
And I stayed with the solution create -> check -> delete files.

Reply:
did you try mocking `fs` instead of reading a file?

Reply:
> did you try mocking fs of fileWriter instead of reading a file?

I don't see the point of mocking here. At the start I run importer and it generates files with mocked input JSON. If I will mock results of importer, these tests only could check if importer was not crashed.

Reply:
you can perform assertions on input parameters in mocks: https://jestjs.io/docs/expect#tohavebeencalledwitharg1-arg2-
On the other hand I don't think it's that important